### PR TITLE
[CAS-945] Docs - MentionListView docusaurus

### PR DESCRIPTION
**Pull request by leandroBorgesFerreira**
[CAS-945](https://stream-io.atlassian.net/browse/CAS-945)

### 🎯 Goal

Documentation for MentionListView

### 🛠 Implementation details

`.md` file Added

### 🎨 UI Changes

Screenshot 2021-04-30 at 14 32 41

### 🧪 Testing

to install docusaurus run:

```
yarn install
```

Then:

Go to documentation folder
Run `yarn run start`
Go to Calling SDK Methods section

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- ~[ ] Changelog is updated with client-facing changes~
- ~[ ] New code is covered by unit tests~
- [x] Comparison screenshots added for visual changes
- [x] Affected documentation updated (CMS, cookbooks, tutorial)
- [x] Reviewers added

### CODE REVIEW

File: documentation/docs/ui/mention_list_view.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+```
+
+| Light Mode | Dark Mode |
+| --- | --- |
+|![light](/img/mentions_list_view_light.png)|![dark](/img/mentions_list_view_dark.png)|
+
+## Adding Mention List View And Binding With View Model
+Here's an example layout containing `MentionListView`:
+```xml
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (adasiewiczr) [inline documentation/docs/ui/mention_list_view.md:31]: It's duplicated 🙈 Maybe let's rename this section to `Binding With View Model` and remove this layout?

Comment (leandroBorgesFerreira) [reply documentation/docs/ui/mention_list_view.md:28]: I added the `Binding With View Model` section, please take a look.

Comment (adasiewiczr) [reply documentation/docs/ui/mention_list_view.md:28]: I think we misunderstood each other. Now we have duplicated `Binding With View Model` section (the second one is in  `Adding Mention List View And Binding With View Model`)
I would rather do something like that:
- Creating Layout (put there `Here's an example layout containing `MentionListView`:` together with XML)
- Binding With View Model (`Just like other views - `MentionListView` comes together with a view model which is responsible for providing necessary data:` )
- Handling Actions

Comment (leandroBorgesFerreira) [reply documentation/docs/ui/mention_list_view.md:13]: Done. But instead of **Create Layout** the section is called **Adding MentionListView**

**Review by adasiewiczr — APPROVED** [review]

**Pull request merged**